Plugins/CoreAPI/server/tests/configuration/getConfigurationResponse.json

```diff
@@ -0,0 +1,508 @@
+{
```

Comment:
File is a duplicate of Burpsuite plugin. Please remove

Reply:
will do